stl/inc/yvals_core.h

```diff
@@ -165,6 +165,7 @@
 // P0608R3 Improving variant's Converting Constructor/Assignment
 // P0616R0 Using move() In <numeric>
 // P0631R8 <numbers> Math Constants
+// P0645R10 <format> text formatting

```

Comment:
We conventionally Title Case Everything, matching the issue title #30:
```suggestion
// P0645R10 <format> Text Formatting
```